src/test/java/io/meeds/qa/ui/steps/definition/HomeStepDefinition.java

```diff
@@ -352,6 +352,12 @@ public void isNoConnectionsBadge() {
                                                 .isTrue();
   }
 
+  @Then("^The spaces badge isn't displayed$")
+  public void thereIsNoSpacesBadge() {
+    assertThat(homeSteps.thereIsNoSpacesBadge()).as("The spaces badge shouldn't be displayed")
```

Comment:
after reflexion, assertThat instruction are more in Homepage.java, for example  checkNumberOfConnectionsInDrawer

So can you move the assertThat instruction at this level ? Can you also modify to isNoConnectionBadge ?

Reply:
Done